src/algorithms/telemetry_decoder/adapters/beidou_b2a_telemetry_decoder.cc

```diff
@@ -0,0 +1,104 @@
+/*!
+ * \file beidou_b2a_telemetry_decoder.cc
+ * \brief Implementation of an adapter of a Beidou B2a NAV data decoder block
+ * to a TelemetryDecoderInterface
+ * \note Code added as part of GSoC 2018 program
+ * \author Dong Kyeong Lee, 2018. dole7890(at)colorado.edu
+ *
+ * -------------------------------------------------------------------------
+ *
+ * Copyright (C) 2010-2015  (see AUTHORS file for a list of contributors)
+ *
+ * GNSS-SDR is a software defined Global Navigation
+ *          Satellite Systems receiver
+ *
+ * This file is part of GNSS-SDR.
+ *
+ * GNSS-SDR is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * GNSS-SDR is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GNSS-SDR. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * -------------------------------------------------------------------------
+ */
+
+
+#include "beidou_b2a_telemetry_decoder.h"
+#include "configuration_interface.h"
+#include "beidou_bcnav2_ephemeris.h"
+#include "beidou_bcnav2_almanac.h"
+//#include "beidou_bcnav2_utc_model.h"
+#include <gnuradio/io_signature.h>
+#include <glog/logging.h>
+
+
+using google::LogMessage;
+
+beidouB2aTelemetryDecoder::beidouB2aTelemetryDecoder(ConfigurationInterface* configuration,
```

Comment:
This is not following the right naming convention. Please check other codes but it should be `BeidouB2aTelemetryDecoder` with capital letter on the `B`. The same error is repeated in the file